### ISSUE TITLE
Force close at first play

### ISSUE THREAD

**Issue by DiamondSpaceGD** [luckydog7/Funkin-android#10]
The game force closed himself after i select a song , or maybe my phone cant handle it?

**Comment by JustTunerYT** [comment]
Same, honor 7a pro. Maybe Phone isn't supported?

**Comment by luckydog7** [comment]
Maybe because low memory but I don't know for sure

**Comment by Just6Alexey** [comment]
also crashes, Xiaomi Redmi 9A

**Comment by Johnsen290** [comment]
> Maybe because low memory but I don't know for sure

yes, that is the problem, since the sprites are so big (4064x4064, that's way too much for a phone to handle), it drains 2 GB RAM just to run one song (I did check how much memory it's using on my PC and yep, it's 2 GB RAM).

**Comment by Johnsen290** [comment]
Get a 3 GB RAM phone.

**Comment by Johnsen290** [comment]
> also crashes, Xiaomi Redmi 9A

The sprites the game's using are so big (4064x4064, that's way too much for a phone to handle, even newer phones)

**Comment by kalebplayz125** [comment]
Ni it's everyone I and some other people can only get hating simulator to work!!

**Comment by noobaccountlol222** [comment]
1: PROBABLY files are missing, maybe try to figure out how to put them back (or make your own?)
2: low space, try clearing some

**Comment by noobaccountlol222** [comment]
![Skid](https://user-images.githubusercontent.com/80225822/110257125-25fd6300-7f7b-11eb-88b1-bacb12869877.png)
spooky

**Comment by pearlbits64** [comment]
I can confirm my phone is and no joke 64 gb which is like good its a Samsung A02 and still doesn't work

**Comment by TerraMKII** [comment]
im using Samsung Galaxy S10 5G and still can't enter ;-;

**Comment by kalebplayz125** [comment]
> I can confirm my phone is and no joke 64 gb which is like good its a Samsung A02 and still doesn't work

lol

**Comment by kalebplayz125** [comment]
> ### [**lol_``_**](

- @ #url

)

**Comment by DanisLixo** [comment]
Im seeing this

**Comment by bloodheroC** [comment]
Wait i have 2gb ram phone and its working and sometimes its crash

**Comment by bloodheroC** [comment]
Even week 6 is workinh

**Comment by MagerRTF** [comment]
> Maybe because low memory but I don't know for sure
try downscaling the sprites, the are so big! (8000x8000)

**Comment by MagerRTF** [comment]
> No it's everyone I and some other people can only get hating simulator to work!!

week 6 crashes too 😞
